apps/sudoku-backend/src/app/sudoku/service/sudoku.service.ts

```diff
@@ -15,21 +15,42 @@ export class SudokuService {
     private readonly sudokuFieldService: SudokuFieldService
   ) {}
 
-  getSudokus(page = 1, take = 25): Promise<SudokuEntity[]> {
-    return this.sudokuRepository.find({
-      relations: ['fields'],
-      skip: take * (page - 1),
-      take,
-    });
+  getSudokus(page = 1, take = 25, type:string): Promise<SudokuEntity[]> {
```

Comment:
so is recht.

Aber was isn bei dir für a formattierung eingstellt? schaut jo schlimm aus